dipy/workflows/tests/test_io.py

```diff
@@ -91,3 +93,33 @@ def test_split_flow():
         split_data, split_affine = load_nifti(split_path)
         npt.assert_equal(split_data.shape, volume[..., 0].shape)
         npt.assert_array_almost_equal(split_affine, affine)
+
+
+def test_convert_sh_flow():
+    with TemporaryDirectory() as out_dir:
+        filepath_in = os.path.join(out_dir, 'sh_coeff_img.nii')
+        filename_out = 'sh_coeff_img_converted.nii'
```

Comment:
I just tested on Windows and I got the same error. 

Not sure what's going on but I had to add the `gz`extension (meaning `sh_coeff_img_converted.nii.gz`) for the test to pass. 

it seems the cleanup between `*.nii` and `*.nii.gz` differ. I need to find out why, but for now, this is your quick fix.

Maybe @arokem has an idea.

Reply:
Happy new year and thank you for debugging on this!